image/templates/CHANGING_CHARTS.md

```diff
@@ -70,3 +70,11 @@ EOM
 # the deploy scripts
 $ ./dev-tools/upgrade-dev-secured-cluster.sh -f test-values.yaml
 ```
+
+## Breaking changes
```

Comment:
Similar to the other PR, please move this up right after the "Notes/tips" section.

Reply:
done